blt/src/Blt/Plugin/Commands/UsersCommands.php

```diff
@@ -37,14 +37,31 @@ public function loadUsers() {
 
     if (array_key_exists('admin', $userConfig)) {
       if ($this->isAdminBlockValid($userConfig['admin'])) {
-        $commands['cgov:user:reset-admin'] = [
-          'users_file_path' => $users_file_path,
-        ];
+        if ($userConfig['admin']['enabled'] === TRUE) {
+          $commands['cgov:user:reset-admin'] = [
+            'users_file_path' => $users_file_path,
+          ];
+        }
+        else {
+          $commands = [
+            'cgov:acsf:block-admin' => [],
+          ];
+          $this->logger->info("Admin has been blocked.");
```

Comment:
I don't love adding `cgov:acsf:block-admin` to the `$commands` variable three separate times. It feels a bit redundant. That said, each instance _is_ for a separate reason _so at least_ the logged messages should reflect it.

1. "Admin user is not explicitly enabled and has been blocked
2. (This ones's good) "Skipping admin user. You must provide both the username and password and set enable to either true or false."
3. "No admin user provided so admin user has been blocked"

Reply:
I have declared it above and i have changed the log messages as listed.